tree/ntupleutil/v7/test/ntuple_inspector.cxx

```diff
@@ -69,6 +69,35 @@ TEST(RNTupleInspector, CompressionSettings)
    EXPECT_EQ("LZMA (level 7)", inspector->GetCompressionSettingsAsString());
 }
 
+// Relevant for RNTuples created with late model extension, see https://github.com/root-project/root/issues/15661 for
+// background.
+TEST(RNTupleInspector, UnknownCompression)
+{
+   FileRaii fileGuard("test_ntuple_inspector_unknown_compression.root");
+   std::vector<float> refVec{1., 2., 3.};
+   {
+      auto model = RNTupleModel::Create();
+
+      auto vecFld = model->MakeField<std::vector<float>>("vecFld", refVec);
+
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "ntuple", fileGuard.GetPath());
```

Comment:
For clarity, perhaps we should set the compress explicitly to 505 through write options.